app/service/CharitiesSectionCompleteService.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package service
+
+import audit.{AuditService, NormalUserAuditEvent}
+import models.requests.OptionalDataRequest
+import models.{AuditTypes, UserAnswers}
+import pages.authorisedOfficials.AuthorisedOfficialsNamePage
+import pages.contactDetails.CharityNamePage
+import pages.otherOfficials.OtherOfficialsNamePage
+import pages.sections._
+import play.api.Logger
+import play.api.libs.json._
+import play.api.mvc.{Call, RequestHeader}
+import repositories.SessionRepository
+import uk.gov.hmrc.http.HeaderCarrier
+import utils.ImplicitDateFormatter
+import viewmodels.authorisedOfficials.AuthorisedOfficialsStatusHelper
+import viewmodels.charityInformation.CharityInformationStatusHelper
+import viewmodels.nominees.NomineeStatusHelper
+import viewmodels.otherOfficials.OtherOfficialStatusHelper
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Success, Try}
+
+@Singleton
+class CharitiesSectionCompleteService @Inject() (
+  sessionRepository: SessionRepository,
+  userAnswerService: UserAnswerService,
+  auditService: AuditService
+) extends ImplicitDateFormatter {
```

Comment:
There is no need to extend ImplicitDateFormatter. Please update and remove its import. Thanks